iFixit/react-commerce
# Refactor product list children cleanup

🚫 blocked by #1373 
🚫 blocked because we wanna check that removing fields from Strapi does not cause any issue in production

closes #1361

This PR is updating the productList model in Strapi to eliminate the childrenHeading field which is not needed anymore 

❗ ❗ ❗ After this PR is merged Strapi should be redeployed in production ❗ ❗ ❗
cc @sterlinghirsh @danielbeardsley 

### QA

1. Navigate to [Strapi](https://refactor-product-list-children-cleanup.govinor.com/admin)
2. Verify that the `childrenHeading` field is not present anymore in the `productList` model
3. Visit [Vercel preview](https://react-commerce-git-refactor-product-list-children-ifixit.vercel.app/Parts)
4. Verify that the product list page still renders correctly
